monitor/linuxmonitor/cgnetcls/cgnetcls.go

```
@@ -170,6 +170,12 @@ func (s *netCls) DeleteCgroup(cgroupname string) error {
 	return nil
 }
 
+// GetAssignedMarkVal -- Gets the mark val assigned to the group
+func GetAssignedMarkVal(cgroupName string) string {
+	mark, _ := ioutil.ReadFile(filepath.Join(basePath, TriremeBasePath, cgroupName, markFile))
```

don’t ignore the error. never ignore any error.